### DISCUSSION TITLE
Only show 2 levels of parents.

### DISCUSSION THREAD

**Discussion by Jaszunai** [SkepticMystic/breadcrumbs#402] [Q&A]
In the grid view that shows at the top of the note, there is a little gear icon. Pressing this icon allows you to set how many levels up the trail you want to see. Is there a way to set a default setting for this at the global level?
I really only want to see up 2 levels of the trail. When I set this manually, it gets reset the next time the index is refreshed.

**Comment by michaelpporter** [comment]
Closing — depth is configurable on v4 codeblocks/views, so you can limit levels. Reopen if that doesn't cover it.
